### PR TITLE
ISPN-15165 RESP PERSIST

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-15165

### CODE REVIEW

File: server/resp/src/main/java/org/infinispan/server/resp/commands/generic/PERSIST.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler,
+                                                      ChannelHandlerContext ctx,
+                                                      List<byte[]> arguments) {
+      byte[] keyBytes = arguments.get(0);
+
+      return handler.stageToReturn(handler.cache().getCacheEntryAsync(keyBytes).thenCompose(e-> {
+         if (e.getLifespan() < 0) {
+            return EXPIRE.NOT_APPLIED;
+         } else {
+            return handler.cache().replaceAsync(e.getKey(), e.getValue(), e.getValue()).thenCompose(__ -> EXPIRE.APPLIED);
```
[the remark this answers is not in the document]
Comment (tristantarrant) [reply server/resp/src/main/java/org/infinispan/server/resp/commands/generic/PERSIST.java]: Done

**Comment by jabolina** [comment]
Sorry, @tristantarrant, I integrated another command before. It needs rebase.

**Comment by tristantarrant** [comment]
Rebased @jabolina

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Thanks, @tristantarrant!

**Pull request merged**